packages/labs/cli/src/lib/commands/labs.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {AbsolutePath} from '@lit-labs/analyzer/lib/paths.js';
+import {Command} from '../command.js';
+
+export const labs: Command = {
+  name: 'labs',
+  description: 'Experimental commands',
+  subcommands: [
+    {
+      name: 'gen',
+      description: 'Generate framework wrappers',
+      options: [
+        {name: 'packageRoot', defaultValue: './'},
```

Comment:
Descriptions would be nice. Even just: "The package to generate wrappers for."

Reply:
Done